Modules/ReportGenerator/Entities/DraggableComponent.php

```diff
@@ -32,4 +33,14 @@ class DraggableComponent extends Model
      * @var string
      */
     protected $connection = 'mysql_report_generator';
+
+    /**
+    * The report formats that belong to the Draggable component.
+    *
+    * @return Response
+    */
+    public function report_formats()
+    {
+        return $this->belongsToMany('ReportFormat')->withTimestamps();
```

Comment:
I like the naming convention for many-to-one, one-to-many elements.  Nice.

Reply:
That's the beauty of Laravel!